doc/source/management/kill_idle_trx.rst

```diff
@@ -9,35 +9,21 @@ engines. If a transaction is idle for more seconds than the threshold
 specified, it will be killed. This prevents users from blocking |InnoDB| purge
 by mistake.
 
-In |Percona Server| :rn:`5.7.17-11` this feature has been re-implemented by
+In |Percona Server| :rn:`5.7.17-11` this feature was re-implemented by
```

Comment:
In this series the doc should only refer to the current implementation

Reply:
The comment not addressed

Reply:
Reference to 5.7 removed